examples/storybooks/sandbox-utils.js

```diff
@@ -0,0 +1,74 @@
+import { getParameters } from 'codesandbox/lib/api/define';
+
+const GIT_URL =
+  'https://api.github.com/repos/fritz-c/react-sortable-tree/contents/';
+
+export const SANDBOX_URL = 'https://codesandbox.io/api/v1/sandboxes/define';
+
+// full url for github api call
+const getURL = filename => `${GIT_URL}/examples/storybooks/${filename}`;
+
+// strip ../../src from the src
+const strip = code => code.replace('../../src', 'react-sortable-tree');
```

Comment:
I used this because `../.../src` was consistent across all storyboards. But its not the best